### ISSUE TITLE
[redis-cluster] Redis cluster does not allow a namespace to be specified

### ISSUE THREAD

**Issue by matty234** [bitnami/charts#8011]
**Which chart**:
Redis-cluster

**Describe the bug**
The redis cluster chart does not include a namespace in the deployment manifests so none is applied when a `helm template <redis chart location> --namespace <namespace>`.

**To Reproduce**

1. Add the Bitnami chart repo
2. Run `helm template bitnami/redis-cluster --namespace=test`

**Expected behavior**
The namespace 'test' is applied to manifests.

**Version of Helm and Kubernetes**:
```
version.BuildInfo{Version:"v3.5.0", GitCommit:"32c22239423b3b4ba6706d450bd044baffdcf9e6", GitTreeState:"dirty", GoVersion:"go1.15.6"}
```

```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.3", GitCommit:"ca643a4d1f7bfe34773c74f79527be4afd95bf39", GitTreeState:"clean", BuildDate:"2021-07-15T21:04:39Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"darwin/arm64"}
Server Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.17-eks-087e67", GitCommit:"087e67e479962798594218dc6d99923f410c145e", GitTreeState:"clean", BuildDate:"2021-07-31T01:39:55Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by rafariossaa** [comment]
Hi,
  Thanks for spotting this out.
  As this is missing in this and other charts, I am creating an internal task to address it. However I can not provide an ETA.
  Meanwhile, pull requests are welcomed, we will be glad to review and merge them.

**Comment by migruiz4** [comment]
Hi @matty234,

We have added the `metadata.namespace` to all the charts in the `bitnami/charts`.

For redis-cluster, it is available since its version `7.2.1` (https://github.com/bitnami/charts/pull/8821), but now we have made it a feature for the entire charts catalog.
